app/src/main/java/emu/skyline/AppDialog.kt

```diff
@@ -72,7 +72,7 @@ class AppDialog : BottomSheetDialogFragment() {
         super.onViewCreated(view, savedInstanceState)
         view.setBackgroundColor(requireContext().getColor(R.color.backgroundColor))
 
-        val missingIcon = ContextCompat.getDrawable(requireActivity(), R.drawable.default_icon)!!.toBitmap(256, 256)
+        val missingIcon = ContextCompat.getDrawable(requireActivity(), R.drawable.default_icon)?.toBitmap(256, 256)
```

Comment:
- If the call to `ContextCompat.getDrawable()` fails, we have bigger problems than the image being null. And even if that wasn't the case, there's no fallback image, so the ImageView would be blank.